gradle.properties

```diff
@@ -1,17 +1,30 @@
 # Project-wide Gradle settings.
+GROUP=com.mapbox.mapboxsdk
 
-# IDE (e.g. Android Studio) users:
-# Gradle settings configured through the IDE *will override*
-# any settings specified in this file.
+POM_URL=https://github.com/mapbox/mapbox-gestures-android
+POM_SCM_URL=https://github.com/mapbox/mapbox-gestures-android
+POM_SCM_CONNECTION=scm:git@github.com:mapbox/mapbox-gestures-android.git
+POM_SCM_DEV_CONNECTION=scm:git@github.com:mapbox/mapbox-gestures-android.git
 
-# For more details on how to configure your build environment visit
-# http://www.gradle.org/docs/current/userguide/build_environment.html
+POM_LICENCE_NAME=The Apache Software License, Version 2.0
```

Comment:
License is actually BSD - https://github.com/mapbox/mapbox-gestures-android/blob/master/LICENSE.md.